### PR TITLE
FetchIndicatorsFromFile - bug fixes and ssl handling

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23918

## Description
1. dropped the SSL warning disable by changing the use of tldextract.
2. Non-indicators in csv/xls files will not be created in case no default_type is assigned.
3. Handled NULL character in csv files - using this: https://stackoverflow.com/questions/7894856/line-contains-null-byte-in-csv-reader-python.

### CODE REVIEW

File: Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py [hunk 1]
```diff
@@ -188,7 +188,9 @@ def detect_type(indicator):
         return FeedIndicatorType.Email
 
     try:
-        if tldextract.extract(indicator).suffix:
+        if tldextract.TLDExtract(cache_file='https://raw.githubusercontent.com/publicsuffix'
```
Comment (ronykoz) [inline Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py:191]: add an explanation here please

File: Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py [hunk 2]
```diff
@@ -6,30 +6,27 @@
...
 def csv_file_to_indicator_list(file_path, col_num, starting_row, auto_detect, default_type, type_col, limit, offset):
     indicator_list = []
 
     # TODO: add run on all columns functionality
 
     line_index = 0
-    with open(file_path) as csv_file:
-        file_reader = csv.reader(csv_file)
+    with open(file_path, 'rU') as csv_file:
+        file_reader = csv.reader(line.replace('\0', '') for line in csv_file)
```
Comment (ronykoz) [inline Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py:19]: add comment here please

**Comment by ronykoz** [comment]
@amshamah419 Could you go over the RN?

File: Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py (hunk 1 above)
Comment (roysagi) [reply Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py:195]: done

**Pull request merged**